.travis.yml

```diff
@@ -6,14 +6,16 @@ node_js:
     - "4.1"
 
 before_install:
+    - rvm install 2.2.2
```

Comment:
@dan-f Does that work? Everything I read said that you had to pick a single language. For example, here's where I took my configuration from:

http://stackoverflow.com/questions/31235146/how-to-run-node-js-and-ruby-tests-within-one-project-on-travis-ci

I'll try your suggestion which will be much cleaner if it works.